x/twasm/contract/callback_msgs.go

```diff
@@ -1,17 +1,26 @@
 package contract
 
+import (
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+)
+
 // TgradeSudoMsg callback message sent to a contract.
 // See https://github.com/confio/tgrade-contracts/blob/main/packages/bindings/src/sudo.rs
 type TgradeSudoMsg struct {
 	PrivilegeChange *PrivilegeChangeMsg `json:"privilege_change,omitempty"`
 
 	BeginBlock *BeginBlock `json:"begin_block,omitempty"`
 	// This will be delivered every block if the contract is currently registered for End Block
-	/// Block height and time is already in Env
+	// Block height and time is already in Env
 	EndBlock *struct{} `json:"end_block,omitempty"`
-	/// This will be delivered after everything.
-	/// The data in the Response is (JSON?) encoded diff to the validator set
+	// This will be delivered after everything.
+	// The data in the Response is (JSON?) encoded diff to the validator set
 	EndWithValidatorUpdate *struct{} `json:"end_with_validator_update,omitempty"`
+
+	// Export dump state for genesis export
+	Export *struct{} `json:"export,omitempty"`
+	// Import genesis state
+	Import *wasmtypes.RawContractMessage `json:"import,omitempty"`
```

Comment:
not json.RawMessage?

Reply:
I want to be consistent with wasmd here. We use this type with other contract messages. it serializes into raw json.